README.md

```diff
@@ -4,14 +4,24 @@
 [![CocoaPods Compatible](https://img.shields.io/cocoapods/v/SwiftGen.svg)](https://img.shields.io/cocoapods/v/SwiftGen.svg)
 [![Platform](https://img.shields.io/cocoapods/p/SwiftGen.svg?style=flat)](http://cocoadocs.org/docsets/SwiftGen)
 
-SwiftGen is a suite of tools written in Swift to auto-generate Swift code (or anything else actually) for various assets of your project:
-
-* [`enums` for your Assets Catalogs images](#uiimage-and-nsimage)
-* [`enums` for your `Localizable.strings` strings](#localizablestrings).
-* [`enums` for your UIStoryboards and their Scenes](#uistoryboard)
-* [`enums` for your NSStoryboards and their Scenes](#nsstoryboard)
-* [`enums` for your Colors](#uicolor-and-nscolor).
-* [`enums` for your Fonts](#uifont-and-nsfont).
+SwiftGen is a tool to auto-generate Swift code for ressources of your projects, to make them type-safe to use.
+
+<table border="0"><tr>
```

Comment:
Ok, GitHub seems to ignore CSS classes so I'm not sure we actually can get rid of them